src/test/java/com/ibm/cloudant/kafka/connect/utils/ConnectorUtils.java

```diff
@@ -145,46 +107,24 @@ public static double stdDeviation(ArrayList<Long> list) {
         return Math.sqrt(sumDiffsSquared / (list.size() - 1));
     }
 
-    public static Properties readTestPropertiesFile() throws IOException {
-        Properties testProperties = new Properties();
-        testProperties.load(new FileReader(new File("src/test/resources/test.properties")));
-        return testProperties;
-    }
-
-    public static Map<String, String> getCommonTestProperties(Properties testProperties) {
+    public static Map<String, String> getTestProperties() {
 
         Map<String, String> connectorProperties = new HashMap<>();
 
-        String systemPropAcct = System.getProperty("cloudant.account");
-        // If a system property was used to specify a test account then use system property account
-        // configuration ahead of the test.properties file and generate a random DB name
-        if (systemPropAcct != null) {
-            // URL, append a random database
-            connectorProperties.put(InterfaceConst.URL, System.getProperty("cloudant.account"));
-            connectorProperties.put(InterfaceConst.DB, "kafka-test-db-" + UUID.randomUUID().toString());
-            // Username
-            connectorProperties.put(InterfaceConst.USER_NAME, System.getProperty("cloudant.user"));
-            // Password
-            connectorProperties.put(InterfaceConst.PASSWORD, System.getProperty("cloudant.pass"));
-        } else {
-            // Fallback to the test.properties file
-            connectorProperties.put(InterfaceConst.URL, testProperties.getProperty(InterfaceConst
-                    .URL));
-            connectorProperties.put(InterfaceConst.USER_NAME, testProperties.getProperty
-                    (InterfaceConst.USER_NAME));
-            connectorProperties.put(InterfaceConst.PASSWORD, testProperties.getProperty
-                    (InterfaceConst.PASSWORD));
+        // some hardcoded properties we don't normally need to override
+        connectorProperties.put(InterfaceConst.DB, "kafka-test-db-" + UUID.randomUUID().toString());
+        connectorProperties.put(InterfaceConst.TOPIC, "test-topic");
+        connectorProperties.put(InterfaceConst.TASKS_MAX, "1");
+        connectorProperties.put(InterfaceConst.BATCH_SIZE, "10000");
+
+        // use provided values from System.getProperties (can also override defaults above)
+        for (String k : CloudantConnectorConfig.CONFIG_DEF.configKeys().keySet()) {

```

Comment:
this looks for anything in the system env which matches one of the config var keys, and sets it. it means you can change auth type easily when running tests from the command line etc